lib/msf/core/exploit/sqli.rb

```diff
@@ -0,0 +1,49 @@
+#
+# This mixin provides helpers to perform SQL injection
+# - provides a level of abstraction for common queries, for example, querying the table names
+# - implements blind and time-based SQL injection in a reusable manner
+# - Highly extendable (user can run any code to perform the requests, encode payloads and parse results)
+#
+module Msf
+  module Exploit::SQLi
+    @@printing_method_names = %i[
+        print_status print_good print_bad print_error print_warning
+        vprint_status vprint_good vprint_bad vprint_error vprint_warning
+    ]
+    def initialize(info = {})
+      super
+      register_advanced_options(
+        [
+          OptFloat.new('SQLI_SLEEP', [ false, 'The delay to sleep on time-based blind SQL injections', 1.0 ])
```

Comment:
Advanced options are registered with SnakeCase
```suggestion
          OptFloat.new('SqliDelay', [ false, 'The delay to sleep on time-based blind SQL injections', 1.0 ])
```